src/Agent/NewRelic/Agent/Core/NewRelic.Agent.Core/AgentManager.cs

```diff
@@ -281,6 +282,7 @@ private void LogInitialized()
                     }
                 }
 
+                Log.Debug($".NET Runtime Version: {RuntimeInformation.FrameworkDescription}");
```

Comment:
Do we need to wrap this in a try catch block? It also looks like that API was only made available begging in `4.7.1`. You might need to test this on a system that just has 4.6.2 installed (and not newer framework versions).

Reply:
I think we're good with the package ref and ILRepack.

Testing on a machine with 4.6.2 only on it is just a bit obnoxious... ;-)

Reply:
I made that comment before the nuget package ref was added. I think that with the nuget package it should be fine.